src/Miso/FFI/Internal.hs

```diff
@@ -508,6 +508,16 @@ focus x = do
 -----------------------------------------------------------------------------
 -- | Fails silently if the element is not found.
 --
+-- Analogous to @document.querySelector('#' + id).select()@.
+select :: MisoString -> JSM ()
+select x = do
+  moduleMiso <- jsg "miso"
+  el <- toJSVal x
+  delay <- toJSVal (50 :: Int)
+  void $ moduleMiso # "callSelect" $ [el,delay]
+-----------------------------------------------------------------------------
+-- | Fails silently if the element is not found.
+--
 -- Analogous to @document.getElementById(id).blur()@
```

Comment:
Can we flip these two comments, so the one you added is below `Analogous to @document.getElementById(id).blur()@`.